docs/modules/clients/pages/java.adoc

```diff
@@ -67,23 +67,75 @@ client.shutdown();
 
 The above code line releases all the used resources and closes connections to the cluster.
 
-=== Java Client Operation Modes
+=== Java Cluster Routing Modes
 
-The client has two operation modes because of the distributed nature of the data and cluster.
+The Java cluster routing mode specifies how the client connects to the cluster.
 
-**Smart Client**: In the smart mode, the clients connect to each cluster member.
+NOTE: In previous releases, this functionality was known as the client operation mode and could be configured as smart or unisocket. 
+If the cluster routing mode is not configured in your client, the configured client operation mode is used.
+
+The mode that you use depends on your network requirements due to the distributed nature of the data and cluster.
+
+In all modes, the following information is provided to the client on initial connection:
+
+* Cluster version
+* Member details, including instance name, IP or UUID, partition group information, CP membership status, and type ({enterprise-product-name} only)
+* Partition group information ({enterprise-product-name} only)
+* CP group leader information ({enterprise-product-name} with `Advanced CP` enabled only) 
+
+The client is updated whenever the cluster version, partition groups, or CP group leader changes.
+
+From {enterprise-product-name} 5.5, you can use any of the following cluster routing modes:
```

Comment:
Did this get addressed somewhere?

Reply:
I added that it can currently only be used with Java clients (I omitted the C# being WIP, as that is not applicable to the current release)

Do we need to say more? I can make it a Note to give more prominence, if you think that would be more visible...

Reply:
This is fine, thanks, I just couldn't find it straight away so wanted to make sure it was covered 👍